webui/react/src/contexts/Store.tsx

```diff
@@ -155,7 +154,7 @@ const initUI = {
   omnibar: { isShowing: false },
   showChrome: true,
   showSpinner: false,
-  theme: themes[BrandingType.Determined][DarkLight.Light],
+  theme: {} as Theme,
```

Comment:
based on the 1st suggestion above, this would just remain as is (prior to this change)

Reply:
does this cause any issues having this as an empty object? Similar question as the branding type as empty string.